### PR TITLE
Task4_ASP.NET_Core

### PR DESCRIPTION
**Pull request by boOhdan**

### CODE REVIEW

File: ASP.NET_Core/FoodOrdering.API/Filters/CustomExceptionFilter.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        }
+
+        public void OnException(ExceptionContext context)
+        {
+            if (!_hostingEnvironment.IsDevelopment())
+            {
+                return;
+            }
+
+            _logger.LogWarning("Failed to connect, retry limit exceeded.", context.Exception);
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/Filters/CustomExceptionFilter.cs]: LogWarning
->
LogError

File: ASP.NET_Core/FoodOrdering.API/TagHelpers/ProductsTagHelpers.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace FoodOrdering.API.TagHelpers
+{
+    public class ProductsTagHelper : TagHelper
+    {
+        public IEnumerable<Product> Products { get; set; }
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagName = "section";
+            var setHtmlContent = string.Empty;
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/TagHelpers/ProductsTagHelpers.cs]: use StringBuilder for setHtmlContent

File: HomeWork3/Program.cs [hunk 3]
```diff
@@ -1,12 +1,8 @@
...
-using System.Text;
-using System.Threading.Tasks;
-
-namespace FoodOrdering
+﻿namespace FoodOrdering.BLL
 {
     public class Program
     {
-        public static async Task Main(string[] args)
+        public static void Main(string[] args)
```
Comment (yevhenbilushko) [inline HomeWork3/Program.cs]: check that it is class lib proj type

Comment (boOhdan) [reply HomeWork3/Program.cs]: Yes, it is

**Review by yevhenbilushko — APPROVED** [review]

**Pull request merged**